common/script/index.js

```diff
@@ -2056,6 +2056,10 @@ api.wrap = function(user, main) {
           cb(null, user);
         }
         return delta;
+      },
+      checkStats: function (req, cb) {
```

Comment:
> "...If there's a smarter way to handle this level checking already, let me know. :)"

I mentioned that this might not be the best approach in the original PR comment; totally open to what you think would work better here. :+1: 

With that said, my reasoning for creating this wrapper is that calling `user.fns.updateStats()` directly never produced the right result, and was causing other issues. I couldn't find another function designed to ensure the current state of a user was correct (such as being lvl1 with 151 exp). This is necessary because `User.set()` does not include a path for checking a user's level. So when `User.set()` is used to set exp, no leveling up occurs, and that's the issue with the initial tour.